src/iotjs_binding.cpp

```diff
@@ -256,7 +256,6 @@ JObject JObject::GetProperty(const char* name) {
       _obj_val.u.v_object,
       reinterpret_cast<const jerry_api_char_t*>(name),
       &res);
-  IOTJS_ASSERT(is_ok);
```

Comment:
I've run into this earlier but couldn't track down why this assert was here and why it's not needed anymore. Actually, it's a blocker now. Any idea why's that?